core/renderers/zelos/path_object.js

```diff
@@ -87,3 +104,75 @@ Blockly.zelos.PathObject.prototype.updateSelected = function(enable) {
     }
   }
 };
+
+/**
+ * @override
+ */
+Blockly.zelos.PathObject.prototype.beginDrawing = function() {
+  this.remainingOutlines_ = {};
+  for (var i = 0, keys = Object.keys(this.outlines_),
+    key; (key = keys[i]); i++) {
+    // The value set here isn't used anywhere, we are just using the
+    // object as a Set data structure.
+    this.remainingOutlines_[key] = 1;
+  }
+};
+
+/**
+ * @override
+ */
+Blockly.zelos.PathObject.prototype.endDrawing = function() {
```

Comment:
No longer an override, here and for `beginDrawing`.

LGTM after you update those comments.